tests/api/test_account.py

```diff
@@ -2699,7 +2699,7 @@ def test_address_update_mutation(
     )
     content = get_graphql_content(response)
     data = content["data"]["addressUpdate"]
-    assert data["address"]["city"] == graphql_address_data["city"]
+    assert data["address"]["city"] == graphql_address_data["city"].upper()
```

Comment:
Why is it upper now?

Reply:
When validating address we are using `google-i18n-address`. This package is changing the address to be valid in a given region or country. For Poland city is made uppercase. We talked with @maarcingebala about this I will try to change this in the next PR.